operator/internal/deploy/webhook_chart.go

```diff
@@ -127,8 +143,14 @@ func installOrRemoveChartOnSKR(ctx context.Context, restConfig *rest.Config, rel
 			return fmt.Errorf("failed to uninstall webhook config: %w", err)
 		}
 		if !uninstalled {
-			//nolint:goerr113
-			return fmt.Errorf("waiting for skr webhook resources to be deleted")
+			return ErrSKRWebhookWasNotRemoved
+		}
+		ready, err := ops.VerifyResources(deployInfo)
+		if err != nil {
+			return fmt.Errorf("failed to verify webhook resources: %w", err)
+		}
+		if ready {
```

Comment:
`ready` or `!ready` ?

Reply:
verify resources returns `ready=true` when resources are there.So I am expecting it to be `false`.